starters/next-react-query-tailwind/src/components/Greeting/Greeting.view.tsx

```diff
@@ -1,15 +1,15 @@
-interface GreetingView {
-  message: string;
-  onClick?: () => void;
-}
+type GreetingView = {
+  message?: string | null;
+  loading?: boolean;
+};
 
-export function GreetingView({ message, onClick }: GreetingView) {
+export function GreetingView({ message, loading }: GreetingView) {
   return (
-    <h1
-      className="text-4xl tracking-tight font-extrabold uppercase text-gray-100 sm:text-5xl md:text-6xl cursor-pointer"
-      onClick={onClick}
-    >
-      {message}
-    </h1>
-  );
+    <div className="flex w-full text-xl justify-center">
+      <div className="mr-[1em]">Message:</div>
+      <div className={`text-left ${loading ? 'grow animate-pulse bg-gray-200 rounded-md' : 'grow-0'}`} role="display-message">
```

Comment:
It would be better to use https://www.npmjs.com/package/classnames here and rewrite this as:
```suggestion
      <div className={cn('text-left', {
        'grow animate-pulse bg-gray-200 rounded-md': loading
        'grow-0': !loading
        })} role="display-message">
```
Slightly easier to read and more semantic with the framework.